team-config.yml

```diff
@@ -77,4 +77,4 @@ tt:
     contact_channel: "#pet"
```

Comment:
line 73 is the issue not the below one but that's a good change to make anyway

Reply:
Ah thanks